pkg/storaged/pages.jsx

```diff
@@ -383,17 +385,18 @@ const ActionButtons = ({ card }) => {
     const buttons = [];
     const items = [];
 
-    if (!card.actions)
+    if (!actions)
         return null;
 
-    for (const a of card.actions) {
-        if (for_menu(a))
-            items.push(make_menu_item(a));
+    for (const action of actions) {
+        if (for_menu(action))
+            items.push(make_menu_item(action));
         else
             buttons.push(
-                <StorageButton key={a.title} onClick={() => a.action(false)}
-                               kind={a.danger ? "danger" : null} excuse={a.excuse}>
-                    {a.title}
+                <StorageButton key={action.title} onClick={() => action.action(false)}
+                               kind={action.danger ? "danger" : null} excuse={action.excuse}
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21927-76c8d326-20250825-170718-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)